src/main/java/com/learntechfree/ltf/category/CategoryService.java

```diff
@@ -0,0 +1,20 @@
+package com.learntechfree.ltf.category;
+
+import com.learntechfree.ltf.entity.Category;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class CategoryService {
+
+    @Autowired
+    private CategoryRepository categoryRepository;
+
+
```

Comment:
We can set guidelines for formatting, 2+ newlines are not preferable.

Reply:
@akasrai  can you please guide for linting guidelines and/or plugin we can use for java code